index.html

```diff
@@ -1,59 +1,76 @@
 <!DOCTYPE html>
-<html>
 
-<title>Personal Project</title>
-<link rel="stylesheet" href="./index.css" type="text/css">
-
-<head>
-    <title>Webpage Demo</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content="A demo page for Mohammed's personal project with Ravine.">
-</head>
-
-<div>
-    <header>
-        <h1 class="centerAlign">Personal Project Demo</h1>
-    </header>
-    
-     <body>
-        
-        <h2 class="centerAlign">Project Description</h2>
-
-        <p class="centerAlign"> This project will focus on creating
-            a e-commerce clothing website in order to showcase
-            and incorporate
-            a wide variety of features from HTML, CSS, and Javascript.
-            Most Notably, I hope to include the following:
-        </p>
-
-        <ul class="centerAlign">
-            <li>Fully Functional Shopping Cart.</li>
-            <li>Payment Method.</li>
-            <li>Shipping, Taxes, and Total Price Calculated.</li>
-            <li>Email Subscription Section.</li>
-            <li>Modern, streamlined style.</li>
-        </ul>
-
-        <p class="centerAlign">
-            After talking with other people within ravine, I finalized
-            a clothing website due to the fact that there are a lot of 
-            different things that can be learned and encorporated into the site.<br>
-            An e-commerce site is also very common within industries and share many elements that can be 
-            used in other sites as well.
-            It fully utilizes HTML, CSS, and Javascript which will<br>
-            give me the ability to get me more up close and personal with
-            web development as a whole.
-        </p>
-
-        <img src="cart.jpeg" style="width: 350px;height: 300px;">
-        <img src="mail.png" style="width: 600px;height: 300px;">
-        <img src="Math.jpeg" style="width: 500px;height: 300px;">
-        <img src="Payments.jpeg" style="width: 900px;height: 300px;">
-    
-    </body>
+<!--Bootstrap link-->
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+<link rel="stylesheet" href="/Users/khanmohammed1/Documents/ravine-projects/mnkhan98.github.io/index.css" type="text/css">
+
+<style>
+  .row div{
```

Comment:
this means all divs in a row will have this css which makes sense now but when u add more rows later its gonna get annoying & messy for you. Try to put a class instead of a div and then add those classes to those divs. so instead of .row div do .row className then add that class name to everything you want